Engines/Wine/Engine/Implementation/script.js

```diff
@@ -120,7 +120,7 @@ module.default = class WineEngine {
             mkdir(this._wineEnginesDirectory + "/runtime");
 
             runtimeJsonFile = new Downloader()
-                .wizard(this._wizard)
+                .wizard(setupWizard)
```

Comment:
I can't see where this `setupWizard` comes from. Is it simply `undefined`?

Reply:
setupWizard is passed to the function in `this.install` using `this.getWizard`

Reply:
Didn't see that. Actually: what's the point of doing that?

Reply:
@plata we have a similar situation with our pre- and post-hooks in the installers. These hooks take both a `wine` parameter and a `wizard` parameter. The `wizard` is kind of redundant because you can fetch it using `wine.wizard()`.

Reply:
It was just for uniformizing. I can replace all of those call with `this.getWizard()`.

Reply:
I think we should remove the `wizard` parameter if it is not required (both in this file and for the hooks).

Reply:
I removed the setupWizard in the function prototype.

Reply:
@ImperatorS79 do you also want to provide a PR where the wizard argument is removed from the installer scripts?

Reply:
This is a lot of work...

Reply:
If you don't have the time that is fine. I couldn't refrain from asking ;)
Alternatively we need an issue where we are reminded about this